src/main/java/nopol_examples/nopol_example_4/NopolExample.java

```diff
@@ -11,6 +11,7 @@ public boolean canBeDividedby3(String a) {
 		int initializedVariableShouldBeCollected = 2;
 		int uninitializedVariableShouldNotBeCollected;
```

Comment:
This line (12) has been identified with a suspiciousness value of 17.96%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_4.NopolExampleTest#test10`
</details>